preCICE-Ubuntu-20.04/install-paraview.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -ex
+
+USER="vagrant"
+
+curl "https://www.paraview.org/paraview-downloads/download.php?submit=Download&version=v5.8&type=binary&os=Linux&downloadFile=ParaView-5.8.1-MPI-Linux-Python3.7-64bit.tar.gz" > /home/vagrant/paraview.tar.gz
+cd /home/vagrant/
+sudo -u ${USER} tar -xzf paraview.tar.gz
+rm paraview.tar.gz
+sudo -u ${USER} ln -sf /home/vagrant/ParaView-5.8.1-MPI-Linux-Python3.7-64bit/bin/paraview /home/vagrant/Desktop/
```

Comment:
You could simplify this to [something like this](https://github.com/precice/precice/blob/22e061e7a2273ce7f2f84a1b930087d005aa6c29/tools/ci/travis-install-dependencies.sh#L64-L75).

Reply:
I updated this, but it is still not perfect. I would prefer to avoid version-specific strings anywhere else apart from the link.

Reply:
Actually the `ParaView-5.8.1-MPI-Linux-Python3.7-64bit/` part is included in the ParaView archive, so I leave it like this for now.